book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :book
```

Comment:
- [ ] Well you can understand that your book needs two essential pieces of information, `title` and `author` and you did great inside the `constructor` of this book, but re-think about the accessing the title and author of the book programmatically, and you will find that you will not able to access the data if you don't have a window to let the user get the information of the book, so that's why please add the `title` and `author` both `getter` and `setter`.